static/app/components/feedback/feedbackItem/feedbackActions.tsx

```diff
@@ -62,6 +64,18 @@ export default function FeedbackActions({
       >
         {feedbackItem.hasSeen ? t('Mark Unread') : t('Mark Read')}
       </Button>
+      {hasSpamFeature && (
+        <Button
+          priority="default"
+          onClick={() => {
+            addLoadingMessage(t('Updating feedback...'));
+            const newStatus = isSpam ? GroupStatus.UNRESOLVED : GroupStatus.IGNORED;
+            resolve(newStatus, mutationOptions);
+          }}
+        >
+          {isSpam ? t('Move to Inbox') : t('Mark as Spam')}
```

Comment:
idk if we're tied to this text, but e.g. gmail uses "Not spam" and "Spam"

Reply:
not tied to the text at all, just what @ryan953  & i came up with - cc @Jesse-Box @jas-kas on wording thoughts